compiler/src/dotty/tools/dotc/typer/ProtoTypes.scala

```diff
@@ -536,14 +536,15 @@ object ProtoTypes {
   /** The result type of `mt`, where all references to parameters of `mt` are
    *  replaced by either wildcards (if typevarsMissContext) or TypeParamRefs.
```

Comment:
The "if typevarsMissContext" part of this comment is incomplete, it was also incomplete before given that we also check `tp.widenExpr.isValueTypeOrWildcard`, so perhaps the comment should just says less:
```suggestion
   *  replaced by either wildcards or TypeParamRefs.
```